### PR TITLE
refactor: deprecate Earn

### PR DESCRIPTION
**Pull request by MarcoWorms**
close https://github.com/yearn/yearn-devdocs/issues/151
close https://github.com/yearn/yearn-devdocs/issues/146

Would be nice to add a yLiquidate description on it's R&D page before merging this PR

### CODE REVIEW

File: docs/resources/r-and-d/yborrow.md [hunk 1]
```diff
@@ -4,6 +4,4 @@ description: Not recommended for retail use
 
 # yBorrow
 
-{% embed url="https://yborrow.finance" caption="" %}
-
-It is an internal tool enabling credit delegation for smart contract to smart contract lending.
+[yBorrow](https://yborrow.finance) is an internal tool enabling credit delegation for smart contract to smart contract lending.
```
Comment (saltyfacu) [inline docs/resources/r-and-d/yborrow.md:7]: Please remove yBorrow. That has evolved into the IronBank

**Comment by saltyfacu** [comment]
We should remove yLiquidate, can you do it?

File: docs/resources/r-and-d/yborrow.md (hunk 1 above)
Comment (MarcoWorms) [reply docs/resources/r-and-d/yborrow.md:7]: nice to know! thanks, will fix

**Comment by MarcoWorms** [comment]
I'll remove yBorrow and yLiquidate as they are kinda loose there and seems to not make sense anymore :)

**Review by KurogeWashu — APPROVED** [review]
gw lgtm

**Comment by saltyfacu** [comment]
great job

**Pull request merged**